src/main/java/monke/tasks/Deadline.java

```diff
@@ -0,0 +1,47 @@
+package monke.tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * The Deadline class represents a task with a specific deadline in the Monke application.
+ * It extends the Task class.
+ */
+public class Deadline extends Task {
+    /** The deadline date and time of the task. */
+    private LocalDateTime date;
```

Comment:
Maybe could indicate time as well since it's not just date 

```suggestion
    private LocalDateTime dateTime;
```